File: speech/src/main/java/net/gotev/speech/SpeechUtil.java
package net.gotev.speech;

import android.content.Context;
import android.content.Intent;
import android.net.Uri;

/**
 * Utility methods.
 * 
 * @author Aleksandar Gotev
 */
public class SpeechUtil {

    /**
     * Opens the Google App page on Play Store
     * @param context application context
     */
    public static void redirectUserToGoogleAppOnPlayStore(Context context) {
        context.startActivity(new Intent(Intent.ACTION_VIEW)
                .setData(Uri.parse("market://details?id=com.google.android.googlequicksearchbox")));
    }
}

File: speech/src/main/java/net/gotev/speech/SpeechUtil.kt
package net.gotev.speech

import android.content.Context
import android.content.Intent
import android.net.Uri

/**
 * Utility methods.
 *
 * @author huuphuoc1396
 */
object SpeechUtil {

    /**
     * Opens the Google App page on Play Store
     * @param context application context
     */
    fun redirectUserToGoogleAppOnPlayStore(context: Context) {
        context.startActivity(Intent(Intent.ACTION_VIEW)
                .setData(Uri.parse("market://details?id=com.google.android.googlequicksearchbox")))
    }
}
